exercises/diffie-hellman/src/lib.rs

```diff
@@ -1,11 +1,11 @@
-pub fn private_key(_p: u64) -> u64 {
-    unimplemented!()
+pub fn private_key(p: u64) -> u64 {
+    unimplemented!("Pick a private key greater than and less than {}", p)
```

Comment:
This is missing something after `greater than`

(Since it would be difficult for a number to be simultaneously greater than `p` and less than `p`, which is what this sentence means if there's nothing after `greater than`)